### PR TITLE
New Recipe: fortune v1.0.0

### PR DESCRIPTION
**Pull request by BeastyBlacksmith**
This is mostly me playing with the wizard.

I think I can delete that patch, no?

### CODE REVIEW

File: F/fortune/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("x86_64", "freebsd"; ),
+    Platform("i686", "windows"; ),
+    Platform("x86_64", "windows"; )
+]
+
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("${bindir}", :gortune)
```
Comment (giordano) [inline F/fortune/build_tarballs.jl:43]:
```suggestion
    ExecutableProduct("gfortune", :gortune)
```

Comment (BeastyBlacksmith) [reply F/fortune/build_tarballs.jl:43]: Hmm... I don't get that path to be honest. When I run that under GH codespaces the binary ends up in `/workspaces/Yggdrasil/F/fortune/build/x86_64-linux-gnu/fIPKCsoP/destdir/bin/`.

Comment (BeastyBlacksmith) [reply F/fortune/build_tarballs.jl:43]: Reading the docstring helps ^^;

File: F/fortune/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/blmayer/gortune.git", "59f27016f1905b381c3f3c748860dadb56aa9ca3")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd gortune/
```
Comment (giordano) [inline F/fortune/build_tarballs.jl:16]:
```suggestion
cd $WORKSPACE/srcdir/gortune/
```

File: F/fortune/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("x86_64", "freebsd"; ),
+    Platform("i686", "windows"; ),
+    Platform("x86_64", "windows"; )
+]
```
Comment (giordano) [inline F/fortune/build_tarballs.jl:38]:
```suggestion
platforms = supported_platforms()
```

**Pull request merged**